### DISCUSSION TITLE
Welcome to Proxmox-Arm64 Discussions!

### DISCUSSION THREAD

**Discussion by jiangcuo** [jiangcuo/Proxmox-Arm64#35] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Raghavek1** [comment]
Hi,
I am trying to install Proxmox Arm edition on Ampere server, I am unable to boot the server with ISO. I am getting lot of errors when i try to add the repo list. Could you please help me to install Proxmox 8.x arm edition on my server ?

Thanks
Raghav
